diff --git a/index.ts b/index.ts
index 4a9bd9232db79458eb26d3c00477d860721c9885..7c3f071064f8fb233966a59153788842a3a10aa6 100644
--- a/index.ts
+++ b/index.ts
@@ -1,4 +1,4 @@
-import { fileURLToPath } from 'node:url'
+// import { fileURLToPath } from 'node:url'
 
 import type { StarlightPlugin } from '@astrojs/starlight/types'
 
@@ -31,16 +31,16 @@ export default function starlightVideosPlugin(): StarlightPlugin {
                 markdown: {
                   rehypePlugins: [[rehypeStarlightVideosTasks]],
                 },
-                vite: {
-                  resolve: {
-                    alias: [
-                      {
-                        find: /^\..*\/Page\.astro$/,
-                        replacement: fileURLToPath(new URL('overrides/Page.astro', import.meta.url)),
-                      },
-                    ],
-                  },
-                },
+                // vite: {
+                //   resolve: {
+                //     alias: [
+                //       {
+                //         find: /^\..*\/Page\.astro$/,
+                //         replacement: fileURLToPath(new URL('overrides/Page.astro', import.meta.url)),
+                //       },
+                //     ],
+                //   },
+                // },
               })
             },
           },
